docs/en/install-upgrade/installing-stack-demo-secure.asciidoc

```diff
@@ -0,0 +1,894 @@
++
+["source","shell"]
+----
+sudo systemctl start kibana.service
+----
++
+{kib} is now configured with the correct fingerprint for {agent} to access {es}. You're now ready to set up {agent}!
+
+[discrete]
+[[install-stack-demo-secure-agent]]
+== Step 7: Install {agent}
+
+Next, we'll install {agent} on another host and use the System integration to monitor system logs and metrics. You can find additional details about these steps in {fleet-guide}/secure-connections.html[Configure SSL/TLS for self-managed Fleet Servers].
+
+. Log in to the host where you'd like to set up {agent}.
+
+. Create a directory for the {es} certificate file:
++
+["source","shell"]
+----
+sudo mkdir /etc/fleet
+----
+. From the first {es} node, copy the `ca.crt` file, and paste it into the `/etc/agent/` directory on the {fleet-server} host. Just to help identify the file we'll also rename it to `es-ca.crt`:
+** `/etc/fleet/es-ca.crt`
+
+. Create a working directory for the installation package:
++
+["source","shell"]
+----
+mkdir elastic-install-files
+----
+
+. Change into the new directory:
++
+["source","shell"]
+----
+cd elastic-install-files
+----
+
+. Open {kib} and go to **Management -> Fleet**.
+
+. On the **Agents** tab, you should see your new {fleet-server} policy running with a healthy status.
+
+. Click **Add agent**. The **Add agent** flyout opens.
+
+. In the flyout, choose an agent policy name, for example `Demo Agent Policy`.
+
+. Leave **Collect system logs and metrics** enabled. This will add the link:https://docs.elastic.co/integrations/system[System integration] to the {agent} policy.
+
+. Click **Create policy**.
+
+. For the **Enroll in Fleet?** step, leave **Enroll in Fleet** selected.
+
+. On the **Install Elastic Agent on your host** step, for this example we select the **Linux Tar** tab, but you can instead select the tab appropriate to the host operating system where you're setting up {fleet-server}.
++
+As with {fleet-server}, note that TAR/ZIP packages are recommended over RPM/DEB system packages, since only the former support upgrading {agent}.
+
+. Run the first three commands one-by-one in the terminal on your {agent} host.
++
+These commands will, respectively:
+
+.. Download the {agent} package from the {artifact-registry}. 
+.. Unpack the package archive.
+.. Change into the directory containing the install binaries.
+
+. Before running the provided `elastic-agent install` command, you'll need to make a few changes:
+
+.. For the `--url` parameter, confirm that the port number is `8220` (this is the default port for on-premises {fleet-server}).
++
+.. Add a `--certificate-authorities` parameter with the full path of your CA certificate file. For example, `--certificate-authorities=/etc/agent/es-ca.crt`.
++
+The result should be like the following:
++
+["source","shell"]
+----
+sudo ./elastic-agent install \
+--url=https://10.128.0.203:8220 \ --enrollment-token=VWCobFhKd0JuUnppVYQxX0VKV5E6UmU3BGk0ck9RM2HzbWEmcS4Bc1YUUM==
```

Comment:
The `enrollment-token` portion of this line should be on its own line with '\' at the end (like the line above it).